src/main/java/graphql/execution/ExecutionStrategyParameters.java

```diff
@@ -3,6 +3,7 @@
 import graphql.PublicApi;
 import graphql.execution.incremental.DeferredCallContext;
 
+import javax.annotation.Nullable;
```

Comment:
Nit: can you please change it to the jetbrains Nullable annotation? We use the jetbrains version in other places